libc/src/__support/freelist_heap.h

```diff
@@ -22,6 +22,9 @@
 
 namespace LIBC_NAMESPACE_DECL {
 
+extern "C" cpp::byte _end;
+extern "C" cpp::byte __libc_heap_limit;
```

Comment:
Is `__libc_heap_limit` already used by other C libraries (in other words is this an existing API)? If not, I'd prefer calling it `__llvm_libc_heap_limit` which matches the naming scheme we use for other symbols that are part of the external ABI.

Reply:
Ah, that was my mistake about the convention. Done.